### PR TITLE
[FormBundle] previous bugfix to avoid empty rows caused another bug

### PR DESCRIPTION
**Pull request by Numkil**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | comma separated list of tickets fixed by the PR

### CODE REVIEW

File: src/Kunstmaan/FormBundle/AdminList/FormSubmissionExportListConfigurator.php [hunk 1]
```diff
@@ -140,8 +140,14 @@ public function buildIterator()
                 'date' => $submission->getCreated()->format('d/m/Y H:i:s'),
                 'language' => $submission->getLang(),
             ];
+            $counter = 1;
             foreach ($submission->getFields() as $field) {
+                $fieldName = $field->getFieldName();
                 $fieldName = preg_replace('/\d+/', '', $field->getFieldName());
+                if(isset($data[$fieldName])) {
+                    $fieldName .= $counter;
```
Comment (acrobat) [inline src/Kunstmaan/FormBundle/AdminList/FormSubmissionExportListConfigurator.php]: Can't you just use the `$field->getSequence()` method?

File: src/Kunstmaan/FormBundle/AdminList/FormSubmissionExportListConfigurator.php [hunk 2]
```diff
@@ -140,8 +140,14 @@ public function buildIterator()
                 'date' => $submission->getCreated()->format('d/m/Y H:i:s'),
                 'language' => $submission->getLang(),
             ];
+            $counter = 1;
             foreach ($submission->getFields() as $field) {
+                $fieldName = $field->getFieldName();
                 $fieldName = preg_replace('/\d+/', '', $field->getFieldName());
```
Comment (acrobat) [inline src/Kunstmaan/FormBundle/AdminList/FormSubmissionExportListConfigurator.php]: Is this line still needed? Or the line above? It's instantly overwritten

**Comment by acrobat** [comment]
@Numkil Also check styleci please

File: src/Kunstmaan/FormBundle/AdminList/FormSubmissionExportListConfigurator.php (hunk 1 above)
Comment (Numkil) [reply src/Kunstmaan/FormBundle/AdminList/FormSubmissionExportListConfigurator.php]: Good point when i look at the database that should work. im gonna test it.

**Pull request merged**